index.html

```diff
@@ -3,13 +3,18 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width" />
+    <link rel="icon" type="image/png" href="static/icon.png"/>    
     <title>Join SYNCS!</title>
     <link rel="stylesheet" href="static/bootstrap.min.css" />
     <link rel="stylesheet" href="static/custom.css" />
   </head>
   <body>
-    <nav class="navbar navbar-dark bg-primary">
-      <a class="navbar-brand" href="https://syncs.org.au">SYNCS</a>
+    <nav class="navbar navbar-dark bg-syncs">
+      <div class="container">
+        <div class="logo">
+          <h1><a href="/"><img src="static/logo-blue.svg" alt="SYNCS" height="32" /></a></h1>
```

Comment:
Any particular reason for the h1?

Reply:
I think the website CSS actually does target h1, which is why I left it like that. I don't see a good reason to do it here given it's not necessary and doesn't make sense semantically either.

Also prettier isn't happy with the HTML as written either. (Turns out PrettierCI doesn't check HTML files by default.)

Reply:
Ahh my work laptop runs prettier on save, mustn't have set that up on home one 😖